controllers/comments.js

```diff
@@ -0,0 +1,21 @@
+const { addCommentsToArticleId, fetchCommentsByArticleId } = require("../models/comments");
+
+exports.getCommentsByArticleId = (req,res,next) => {
+    const {article_id} = req.params;
+  
+    fetchCommentsByArticleId(article_id).then((result) => {
+        res.status(200).send({comments:result});
+    }).catch((err) => {
+        next(err);
+    });
+};
+
+exports.postCommentsToArticleId = (req,res,next) => {
+    const {article_id} = req.params;
+    const newComment = req.body;
+    addCommentsToArticleId(article_id,newComment).then((result) => {
+        res.status(201).send({comment:result});
```

Comment:
Excellent that you're returning the data on an appropriate key, makes it easier to identify and work with.